modules/fscloud/README.md

```diff
@@ -0,0 +1,48 @@
+# Financial Services Cloud Profile
+
+This is a profile for Event Notifications that meets Financial Services Cloud requirements.
+It has been scanned by [IBM Code Risk Analyzer (CRA)](https://cloud.ibm.com/docs/code-risk-analyzer-cli-plugin?topic=code-risk-analyzer-cli-plugin-cra-cli-plugin#terraform-command) and meets all applicable goals.
+
```

Comment:
The usage is missing

Reply:
@ocofaigh I am not seeing usage section in other TIM FScloud modules, are we supposed to add for others as well?

Reply:
I also noticed that pre-commit is removing the usage section automatically from fscloud module readme doc.

Reply:
The usage is not automated - it has to be manually added

Reply:
Every module (and submodule) must have a usage block. If missing from other modules, that needs to be addressed